docs/ai/conceptual/agents.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Let Agents and Copilots Bring Functionality to Your App"
+description: "Learn how agents and copilots intelligently extend the functionality of LLMs to efficiently meet user goals in .NET."
+author: catbutler
+ms.topic: concept-article #Don't change.
+ms.date: 04/15/2024
+
+#customer intent: As a .NET developer, I want to understand how agents and copilots extend the functionality of AI apps, so that my apps can handle any type of content and meet more user goals.
+
+---
+
+# How agents and copilots work with LLMs
+
+Agents and copilots both extend an LLM's capabilities by intelligently invoking external functionality, such as sending an email.
+
+- **An agent** is an artificial intelligence that can answer questions and automate processes for users. Agents can determine which functions will meet a user's goal, and then call those functions on the user's behalf.
+- **A copilot** is a type of agent that works side-by-side with a user. Unlike an agent, a copilot isn't fully automated&mdash;it relies on user interaction. A copilot can help a user complete a task by providing suggestions and recommendations.
```

Comment:
```suggestion
- A *copilot* is a type of agent that works side-by-side with a user. Unlike an agent, a copilot isn't fully automated&mdash;it relies on user interaction. A copilot can help a user complete a task by providing suggestions and recommendations.
```

Reply:
See https://learn.microsoft.com/en-us/contribute/content/text-formatting-guidelines#italics for use of bold vs. italics.